tests/unit/test_io_hdf5_h5tools.py

```diff
@@ -739,12 +742,12 @@ def test_copy_h5py_dataset_h5dataio_input(self):
                              self.f['test_copy'][:].tolist())
 
     def test_list_fill_empty(self):
-        dset = self.io.__list_fill__(self.f, 'empty_dataset', [], options={'dtype': int, 'io_settings': {}})
+        dset = self.io.__list_fill__(self.f, 'empty_dataset', [], True, options={'dtype': int, 'io_settings': {}})
```

Comment:
Can we add:
```
self.assertEqual(get_data_shape(dset), (None,))
```
to confirm that this dataset is expandable?

Reply:
We already test `list_fill` expandability with the other tests. We can add it here to, but I think it is redundant. (which is why I didn't add it).

Reply:
That's fair. Redundancy in the tests can be nice to catch unexpected edge cases, but in this case, after looking at the code again, I agree, it's not needed.